meshnesh/android-level-up
# #156005187 Handling configuration changes

# What does this PR do?

Given when a user rotates the screen of my app to simulate a configuration change. The page should not refresh and that state is maintained displaying the data that had already loaded.

## How should this be manually tested?

* Clone the repo.
* Open the project in android studio.
* Run the project using an emulator or adb. (preferred Tablet device)
* Enable auto rotate in the settings.
* Rotate the emulator to see the change.

## What are the relevant pivotal tracker stories?

[#156005187](https://www.pivotaltracker.com/story/show/156005187)
